internal/scaffold/helm/entrypoint.go

```diff
@@ -37,5 +37,5 @@ func (e *Entrypoint) GetInput() (input.Input, error) {
 const entrypointTmpl = `#!/bin/sh -e
 
 cd $HOME
-exec ${OPERATOR} exec-entrypoint helm --watches-file=$HOME/watches.yaml $@
+exec ${OPERATOR} --watches-file=$HOME/watches.yaml $@
```

Comment:
If the goal is to remove the `exec-entrypoint`, then;

- If we will remove the `execentrypoint` then, should not the operator-sdk run command start to  call the `cmd/<type>-operator/main.go` instead of it? E.g `go run cmd/ansible-operator/main.go`
- Also, is not this PR missing the fragment with the explanation that `exec-entrypoint` was removed and how users can achieve the same goal after this pr?. 
- And then, should we not remove the dir `cmd/operator-sdk/execentrypoint`?

Reply:
> If we will remove the execentrypoint then, should not the operator-sdk run command start to call the cmd/<type>-operator/main.go instead of it? E.g go run cmd/ansible-operator/main.go (see that both codes are duplicated now)

`go run cmd/ansible-operator/main.go` has two big problems:
1. It would require that source code to exist on the end-user's machine.
2. It would require Go to be installed on the end-user's machine. The Helm and Ansible operators do not (and should not) require users to install Go.

Either way, what you're suggesting is pretty much what happens right now. All of the shared code is in the helm and ansible `Run` functions. So now, `run local` and the binaries will be running the same code, just with slightly different setup code.

> Also, is not this PR missing the fragment with the explanation that exec-entrypoint was removed and how users can achieve the same goal after this pr?.

:+1: 

> And then, should we not remove the dir cmd/operator-sdk/execentrypoint?

:+1:

Reply:
Thanks @camilamacedo86 and @joelanford. Will make these changes.
Just a small clarification. Are we going to deprecate `exec-entrypoint` in the upcoming release (after 0.19.0, and remove later) or removing it directly along with this PR?.

Reply:
if we are removing from the Dockerfile scaffolded than,  I understand that we should remove the `cmd/operator-sdk/execentrypoint` direct in this case. @joelanford wdyt?

Reply:
Yeah, I think we can go ahead and remove it in this PR. 

This is a hidden, undocumented CLI subcommand that is only meant to be used in the base image. And with this PR, that subcommand is no longer needed.